src/classes/Cart.ts

```diff
@@ -215,7 +215,7 @@ abstract class Cart {
 
         this.offer.data('handleTimestamp', moment().valueOf());
 
-        this.offer.setMessage('Powered by TF2 Automatic');
+        this.offer.setMessage('Powered by TF2 Automatic. ' + process.env.OFFER_MESSAGE || 'Powered by TF2 Automatic');
```

Comment:
If there is no message, it will say `Powered by TF2 Automatic. Powered by TF2 Automatic`

Reply:
I've tried put the brackets before, but this CLI checking things rejected it. That one I've tried and done have that problem.
![checkerror](https://user-images.githubusercontent.com/47635037/75480579-7fcc6400-59dc-11ea-85f9-3ee6ce5700a3.PNG)

Reply:
You are not using prettier, just use that and it will be fixed